tests/test_client.py

```diff
@@ -523,6 +524,72 @@ def test_jsapi_get_jsapi_card_params(self):
             signature_dict,
         )
 
+    def test_jsapi_card_ext(self):
+        card_ext = JsapiCardExt(openid="2", signature="asdf")
+        assert "outer_str" not in card_ext.dict()
```

Comment:
一般来说单元测试中我们会用 `TestCase.assert` 系列方法，
类似的阐述在 StackOverflow 上有: https://stackoverflow.com/questions/6361147/why-do-assertions-in-unittest-use-testcase-assertequal-not-the-assert-keyword

```suggestion
        self.assertNotIn("outer_str", card_ext.dict())
```

Reply:
好的, 我以前用[pytest](https://docs.pytest.org/en/latest/), 文档里给的示例是`assert`, 我以为unittest也是这样的

Reply:
pytest 用 assert 没关系的，我们这里 unittest 也是用 pytest 跑的，可以不用改。